client-misk-dynamodb/src/main/kotlin/app/cash/backfila/client/misk/dynamodb/DynamoDbBackfillModule.kt

```diff
@@ -20,6 +20,9 @@ class DynamoDbBackfillModule<T : DynamoDbBackfill<*, *>> private constructor(
 ) : KAbstractModule() {
   override fun configure() {
     install(DynamoDbBackfillBackendModule)
+    // Ensures that the backfill class has an @Inject annotation and
```

Comment:
"Ensures that the backfill class is injectable. If you are failing this check you probably want to add an `@Inject` annotation to your class or check that all of your dependencies are provided."

I like it. I only propose the edit above since there are many ways your service might have to inject the Backfill.

Reply:
Right, makes sense to me. Updated!